### PR TITLE
[OPPRO-176] Unify cross-JNI data transmission model for backends (Patch 2)

### PR DESCRIPTION
**Pull request by zhztheplayer**

### CODE REVIEW

**Comment by zhztheplayer** [comment]
@zhouyuan @rui-mo @jinchengchenghh Can any of you please help me review this, thanks.

File: backends-velox/src/main/java/io/glutenproject/columnarbatch/ArrowColumnarBatches.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
+package io.glutenproject.columnarbatch;
```
Comment (zhouyuan) [inline backends-velox/src/main/java/io/glutenproject/columnarbatch/ArrowColumnarBatches.java:1]: Nit: Adding the ASF header?

Comment (zhztheplayer) [reply backends-velox/src/main/java/io/glutenproject/columnarbatch/ArrowColumnarBatches.java:18]: done

**Comment by rui-mo** [comment]
Verified locally on TPC-DS.

**Review by rui-mo — APPROVED** [review]

**Pull request merged**